packages/client/src/app/shared/alerts/components/alerts/alerts.component.ts

```diff
@@ -0,0 +1,30 @@
+import { Component, OnInit, Inject } from '@angular/core';
+import { MAT_SNACK_BAR_DATA } from '@angular/material/snack-bar';
+import { AlertsStateModel, AlertType } from '../../models';
+
+@Component({
+    selector: 'alerts',
+    templateUrl: './alerts.component.html',
+    styleUrls: ['./alerts.component.less']
+})
+export class AlertsComponent implements OnInit {
+    public alertType = AlertType;
+
+    constructor(@Inject(MAT_SNACK_BAR_DATA) public data: AlertsStateModel) {}
+
+    ngOnInit(): void {
+        switch (this.data.type) {
+            case AlertType.Success:
+                document.documentElement.style.setProperty('--alert-color', 'rgb(60,128,40)');
+                break;
+            case AlertType.Error:
+                document.documentElement.style.setProperty('--alert-color', '#DD4C4F');
+                break;
+            case AlertType.Info:
+                document.documentElement.style.setProperty('--alert-color', 'rgb(80,154,233)');
+                break;
+            case AlertType.Warning:
+                document.documentElement.style.setProperty('--alert-color', '#d9a107')
```

Comment:
Can these be consistent in their color formatting? Either `rbg()` or hex values for all of them.

Reply:
i'll make them all rgb then